e2e/nomostest/iam/account.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iam
+
+import (
+	"fmt"
+
+	"kpt.dev/configsync/e2e"
+	"kpt.dev/configsync/e2e/nomostest"
+)
+
+// ValidateServiceAccountExists validates that the specified GSA exists in the
+// test project.
+func ValidateServiceAccountExists(nt *nomostest.NT, gsaEmail string) error {
```

Comment:
We can also validate if the SA has the expected access.

Reply:
I'm not entirely sure how to do that. Would you like to do that in a follow up PR?

Reply:
FWIW, I didn't add this check, just refactored it into a function, since it was being used multiple places.

Reply:
ACK